src/app/components/Events/events.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+class events extends React.Component {
+
+   constructor(props) {
+      super(props);
+
+      this.state = {
+         prop1: 'Initial data...',
+         prop2: 0
+      }
+      this.updateStatus = this.updateStatus.bind(this);
+      this.buttonEventCheck = this.buttonEventCheck.bind(this);
+   };
+
+   updateStatus(e) {
+      // this method will invoked by 'myPropTwo' property which is called by child form in 
+      // onChange event
+      this.setState({ prop1: e.target.value });
+   }
+   buttonEventCheck() {
+      this.state.prop2 = this.state.prop2 + 1;
+      this.setState({ prop2: this.state.prop2 });
+   }
+   render() {
+      return (
+         <div>
+            <ChildComponent myPropOne={this.state.prop1} myPropTwo={this.updateStatus}
+               propEvent={this.buttonEventCheck} proEventValue={this.state.prop2} />
```

Comment:
can you please add example for how we can pass parameter to buttonEventCheck function on button click

Reply:
ok

Reply:
i have taken here onChangeEvent instead onClick

Reply:
button click event with parameter has called up